composer.json

```diff
@@ -10,10 +10,10 @@
     ],
     "require-dev": {
         "phake/phake": "*",
-        "symfony/form": "^2.3|^3.0",
+        "symfony/form": "^2.3|^3.0|^4.0",
         "jms/serializer": "*",
-        "symfony/security-core": "^2.6|^3.0",
-        "symfony/framework-bundle": "^2.6|^3.0",
+        "symfony/security-core": "^2.6|^3.0|^4.0",
+        "symfony/framework-bundle": "^2.6|^3.0|^4.0",
         "sensio/framework-extra-bundle": "~3.0",
```

Comment:
Could be changed to: `"sensio/framework-extra-bundle": "~3.0|^4.0|^5.0",` to bump `symfony/dependency-injection` on version 4 😉